rllib/models/torch/modules/reshape.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+from ray.rllib.utils.framework import try_import_torch
+
+torch, nn = try_import_torch()
+
+
+class Reshape(nn.Module):
```

Comment:
Can we add this to the file where the SlimFC stuff is located instead?

Reply:
+1

Reply:
done